discordrb.json

```diff
@@ -57,6 +57,18 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "supportsUserLogin": "yes",
```

Comment:
Not necessarily - the feature is still useful for non-2FA accounts. While tokens are preferred, I would still include this field as there is a valid use case.